### ISSUE TITLE
Timestamp validation 

### ISSUE THREAD

**Issue by arnulfojr** [segmentio/loosely-validate-event#8]
If the timestamp is manually passedthe validation fails.

https://github.com/segmentio/loosely-validate-event/pull/7

**Comment by nickcoutsos** [comment]
I'm having the same problem. My use-case involves receiving ISO8601 date strings (which Segment accepts) but this validation library only accepts parsed `Date` objects which is unhelpful. I'm using this library through Segment's _analytics-node_ so it's not something I can circumvent.

**Comment by musashi-glitch** [comment]
yes, the timestamp validation here is expecting a `date` type which, to my knowledge, doesn't exist in JS? now I can't backfill my data :/

**Comment by arnulfojr** [comment]
I’m afraid the owners are not putting attention to this repo :/ the issue and PR have been around very long without any feedback...
